stream-chat-android-state/src/main/java/io/getstream/chat/android/state/plugin/logic/channel/internal/ChannelLogic.kt

```diff
@@ -696,7 +697,8 @@ internal class ChannelLogic(
             is NotificationChannelMutesUpdatedEvent -> event.me.channelMutes.any { mute ->
                 mute.channel?.cid == mutableState.cid
             }.let(channelStateLogic::updateMute)
-
+            is UserMessagesDeletedEvent ->
+                channelStateLogic.deleteMessagesFromUser(event.user.id, event.hardDelete, event.createdAt)
```

Comment:
Can you use parameter names there?
So that we can see that `createdAt` is used as `deletedAt`